prow/cmd/hook/githubagent.go

```diff
@@ -74,8 +76,10 @@ func (ga *GitHubAgent) handlePullRequestEvent(pr github.PullRequestEvent) {
 		"url":  pr.PullRequest.HTMLURL,
 	})
 	l.Infof("Pull request %s.", pr.Action)
-	if err := ga.prTrigger(pr); err != nil {
-		l.WithError(err).Error("Error triggering after pull request event.")
+	if ga.Plugins.Enabled(pr.PullRequest.Base.Repo.FullName, triggerPluginName) {
```

Comment:
You have trigger here, but trigger AND lgtm below. Which means that you need to know what methods each plugin is interested in. I'd love to have the list of plugins be a list (rather than hard-coded individual variables), and some sort of registration (including what methods it's interested in)